MovieChecker/MovieChecker/BoxOfficeViewController.m

```diff
@@ -0,0 +1,33 @@
+//
+//  BoxOfficeViewController.m
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/21/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "BoxOfficeViewController.h"
+
+@interface BoxOfficeViewController ()
+
+@end
+
+@implementation BoxOfficeViewController
+
+#pragma mark - Table view data source
```

Comment:
I know this came from Xcode with the table view controller template, but if you change this to `#pragma mark - UITableViewDataSource methods`, then you can option- or command-click on `UITableViewDataSource` to see the mini-docs or jump to the definition, respectively.